src/components/IKUpload/props.ts

```diff
@@ -2,6 +2,47 @@ import { UploadResponse } from 'imagekit-javascript/dist/src/interfaces';
 import PropTypes, { InferProps } from 'prop-types';
 import React from 'react';
 
+type TransformationObject = {
+  type: "transformation";
+  value: string;
+};
+
+type GifToVideoOrThumbnailObject = {
+  type: "gif-to-video" | "thumbnail";
+  value?: string;
+};
+
+type AbsObject = {
+  type: "abs";
+  value: string;
+  protocol: "hls" | "dash";
+};
+
+type PostTransformation = TransformationObject | GifToVideoOrThumbnailObject | AbsObject;
+
+type TransformationType = {
+  pre?: string;
+  post?: PostTransformation[];
+};
+
+interface BgRemoval {
+  name: string
+  options: {
```

Comment:
options is not required as per docs https://docs.imagekit.io/extensions/overview/background-removal